starknet_py/net/schemas/gateway.py

```diff
@@ -85,13 +88,15 @@ class TransactionSchema(Schema):
 
 
 class InvokeTransactionSchema(TransactionSchema):
-    contract_address = Felt(data_key="contract_address", required=True)
+    contract_address = Felt(data_key="contract_address", load_default=None)
+    sender_address = Felt(data_key="sender_address", load_default=None)
     calldata = fields.List(Felt(), data_key="calldata", required=True)
     entry_point_selector = Felt(data_key="entry_point_selector", load_default=None)
     nonce = Felt(data_key="nonce", load_default=None)
 
     @post_load
     def make_dataclass(self, data, **kwargs) -> InvokeTransaction:
+        replace_invoke_sender_address_with_contract_address(data)
```

Comment:
Shouldn't we replace contract_address with sender_address?

Reply:
It's an mistake in the function name - it does replace contract address with sender.